src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -882,11 +882,10 @@ public final void testExecuteLogOutput() throws Exception {
         final String auditFinishedMessage = bundle.getString(DefaultLogger.AUDIT_FINISHED_MESSAGE);
 
         final List<MessageLevelPair> expectedList = Arrays.asList(
-                new MessageLevelPair("checkstyle version .*", Project.MSG_VERBOSE),
                 new MessageLevelPair("Adding standalone file for audit", Project.MSG_VERBOSE),
                 new MessageLevelPair("To locate the files took \\d+ ms.", Project.MSG_VERBOSE),
-                new MessageLevelPair("Running Checkstyle  on 1 files", Project.MSG_INFO),
                 new MessageLevelPair("Using configuration file:.*", Project.MSG_VERBOSE),
+                new MessageLevelPair("Running Checkstyle (version .*) on 1 files", Project.MSG_INFO),
```

Comment:
`Running Checkstyle version 5.0.1 on 11 files`
`Running Checkstyle v. 5.0.1 on 11 files`
`Running Checkstyle of version 5.0.1 on 11 files`